src/helpers.js

```diff
@@ -25,14 +26,42 @@ const getCourseImageUrl = courseData => {
     : "images/course_image.jpg"
 }
 
-const getCourseNumber = courseData => {
-  let courseNumber = courseData["sort_as"]
-  if (courseData["extra_course_number"]) {
-    if (courseData["extra_course_number"]["sort_as_col"]) {
-      courseNumber = `${courseNumber} / ${courseData["extra_course_number"]["sort_as_col"]}`
+const getDepartments = courseData => {
+  const primaryDepartmentNumber = courseData["sort_as"].split(".")[0]
+  const department = departmentsJson.find(departmentObject => {
+    return departmentObject["depNo"] === primaryDepartmentNumber
+  })
+  if (department) {
+    let departments = [department["title"]]
+    if (courseData["extra_course_number"]) {
+      departments = departments.concat(
+        courseData["extra_course_number"].map(extraCourseNumber => {
+          const extraDepartmentNumber = extraCourseNumber[
+            "linked_course_number_col"
+          ].split(".")[0]
+          const department = departmentsJson.find(departmentObject => {
+            return departmentObject["depNo"] === extraDepartmentNumber
+          })
```

Comment:
we do this same thing above on line 31-33 - would it make sense to have a little function like this?

```js
getDepartment = departmentNumber => ( ... )
```

Reply:
All set.